### ISSUE TITLE
multilib/multiarch runtime selection on Linux

### ISSUE THREAD

**Issue by rpavlik** [KhronosGroup/OpenXR-SDK-Source#142]
cc @Ralith 

Can't really load a 64-bit so into a 32-bit process, right?

**Comment by Ralith** [comment]
Yeah, though if the loader keeps searching when dlopen returns an error it should Just Work.

**Comment by rpavlik** [comment]
cc @alee-ccu

**Comment by rpavlik** [comment]
@Ralith so you'd just install different file names for the 32-bit and 64-bit manifest so they both can coexist?

Hmm - but they can't both be active_runtime.json

**Comment by Ralith** [comment]
Oh, right, I was thinking in Vulkan loader terms. Yeah, this is a problem;
probably the json format will need to be changed. How is it handled on
Windows?

**Comment by brycehutchings** [comment]
Windows has separate registry keys for 32bit and 64bit. A Linux equivalent solution would be different well known file names or different well known file paths for the json. IMO, Ideally we keep the json schema unchanged.

**Comment by Ralith** [comment]
Updating the schema has advantages: it allows XR_RUNTIME_JSON to work for
all processes, and makes it impossible to forget to update one of multiple
files. I don't think the amount of breakage would be much different, either.

**Comment by rpavlik** [comment]
well, but it would introduce a difference between platforms: right now, there's only one runtime manifest parser used all over.

**Comment by rpavlik** [comment]
(I'm in favor of either decorated active_runtime.json filename, or a pattern of active_runtime.json files and it tries until it succeeds in loading from one of them)

**Comment by Ralith** [comment]
I suppose we could make XR_RUNTIME_JSON a delimited list, too, and there
might be other applications for trying multiple runtimes.
